packages/mtr.rb

```diff
@@ -0,0 +1,34 @@
+require 'package'
+
+class Mtr < Package
+  description "mtr combines the functionality of the 'traceroute' and 'ping' programs in a single network diagnostic tool."
+  homepage 'https://www.bitwizard.nl/mtr/'
+  version '0.93'
+  source_url 'ftp://ftp.bitwizard.nl/mtr/mtr-0.93.tar.gz'
```

Comment:
Use Github releases instead of FTP for the download. https://github.com/traviscross/mtr/archive/v0.93.tar.gz

Reply:
One exception to using the github releases is if the ftp site version doesn't require any extra steps such as autogen.sh, autoconf or automake.  There is also https://www.bitwizard.nl/mtr/files/ but it doesn't look like the latest release is available.

Reply:
Indeed, in the github archive there is no configure script. So I let it that way.